common/changes/users-navneetg-fixcalloutdefaultgap_2017-04-19-23-18.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Updating the default gapSpace for CallOut and Tooltip to be zero rather than 16",
```

Comment:
please follow the "Component: description" format, maybe "Callout/Tooltip: Updated default gapSpace from 16 to 0"